app/javascript/pulfalight/toc.es6

```diff
@@ -43,8 +43,14 @@ export default class TocBuilder {
   setupEventHandlers() {
     // Listen for click on the leaf node and follow link to component
     this.element.on('activate_node.jstree', (e, data) => {
-      let location = `/catalog/${data.node.id}?onlineToggle=${this.toggleElement.checked}`
-      window.location = location
+      // Only handle left-clicks (button 0)
+      // This allows right-clicks and middle-clicks to use the default behavior of the links
+      if (e.originalEvent && e.originalEvent.button === 0) {
```

Comment:
This code change reflects some imprecise text written in the ticket and breaks the regular functionality of the ToC component. Reversing changes makes everything work as expected.

```
those should be real links, even if our javascript is capturing them, so that middle clicks work.
```